app/lib/dttp/params/placement_assignment.rb

```diff
@@ -118,6 +119,14 @@ def training_initiative_param
         }
       end
 
+      def study_mode_params
+        return {} if trainee.study_mode.blank?
+
+        {
+          "dfe_studymodeid@odata.bind" => "/dfe_studymodeses(#{course_study_mode_id(trainee.study_mode)})",
```

Comment:
If this is an `odata.bind` i think the format of the params needs to be `dfe_StudyModeId`